packages/app/src/app/store/modules/dashboard/computed.js

```diff
@@ -17,11 +17,16 @@ export function getFilteredSandboxes(sandboxes) {
     }
 
     if (orderField === 'title') {
-      return s.title || s.id;
+      return s.title || s.alias || s.id;
     }
 
     return s[orderField];
-  }).filter(x => blacklistedTemplates.indexOf(x.source.template) === -1);
+  }).filter(
+    x =>
+      blacklistedTemplates.indexOf(
```

Comment:
@CompuIves Instead of using `indexOf === -1`, you could use `!includes`

```js
x => ! blacklistedTemplates.includes(x.forkedTemplate ? x.forkedTemplate.id : x.source.template)
```